ykrt/src/mt.rs

```diff
@@ -407,50 +438,69 @@ mod tests {
         let mut mtt = MTBuilder::new().hot_threshold(2).init();
 
         // The program is silly. Do nothing twice, then start again.
-        enum ByteCode {
-            Nop,
-            Restart,
-        }
-        let prog = vec![ByteCode::Nop, ByteCode::Nop, ByteCode::Restart];
+        const NOP: u8 = 0;
+        const RESTART: u8 = 1;
+        let prog = vec![NOP, NOP, RESTART];
 
         // Suppose the bytecode compiler for this imaginary language knows that the first bytecode
         // is the only place a loop can start.
         let locs = vec![Some(Location::new()), None, None];
 
         struct IO {
-            prog: Vec<ByteCode>,
+            prog: Vec<u8>,
             pc: usize,
+            count: usize,
         }
 
-        let interp_step = |mut tio: IO| {
+        #[interp_step]
+        fn dumb_interp_step(tio: &mut IO) {
+            // FIXME make `inputs` a struct. Named fields would be much nicer.
```

Comment:
Kill this line. `tio` is already a struct.

Reply:
Fixed in 806a168.